packages/rancher-pushprox/charts/values.yaml

```diff
@@ -52,6 +54,15 @@ clients:
     keyFile: ""
     caCertFile: ""
 
+  # By default, the client is a DaemonSet, but if the component's pods run on some, not all, nodes,
+  # the client should be a Deployment which pods run on the nodes where the component's run.
+  deployment:
+    # The number of pods the client Deployment should have. 0 means the client is a DaemonSet.
+    replicas: 0
+    # Options to select nodes based on nodeAffinity, inter-pod affinity and anti-affinity.
+    # This can not be empty when replicas is not 0.
+    affinity: {}
```

Comment:
Can a DaemonSet specify `affinity` too? Ideally this should be located in the section where nodeSelectors and tolerations are described.